components/Comment/hooks/useQuillContent.ts

```diff
@@ -1,29 +1,45 @@
-import { useEffect, useRef, useState } from "react";
-import Quill from 'quill';
+import { useEffect, useState } from "react";
+import Quill from "quill";
+import { isEmpty } from "~/config/utils/nullchecks";
 
 type Args = {
-  quill: Quill | undefined,
-  content: object,
-}
+  quill: Quill | undefined;
+  content: object;
+};
 
 const useQuillContent = ({ quill, content = {} }: Args) => {
-
   const [_content, _setContent] = useState<object>(content);
-  const contentRef = useRef<object>(_content);
+  const [isInitialized, setIsInitialized] = useState<boolean>(false);
```

Comment:
instead of using a state object here, maybe it's better to use a ref

```
const isInitialized = useRef(false)`

...
if (!isInitialized.current) {
...
}
isInitialized.current = true;
```